src/clj/witan/send/model/data_products.clj

```diff
@@ -187,29 +156,28 @@
   (let [by-state (future (summarise-results
                           (partial roll-up-calendar-year-by-state
                                    (states/calculate-valid-states-from-setting-academic-years valid-states))
-                          simulations))
+                          simulations)) ;; FIXME - this should be a state filter for all data products
         total-in-send-by-ay (future (summarise-results roll-up-calendar-year-by-ay simulations))
-        total-in-send (future (into []
-                                    (map :total-population)
-                                    (summarise-results roll-up-calendar-year-by-total-send-population simulations)))
-        total-in-send-by-need (future (summarise-results roll-up-calendar-year-by-total-in-send-by-need simulations))
-        total-in-send-by-setting (future (summarise-results roll-up-calendar-year-by-total-in-send-by-setting simulations))
+        total-in-send-by-ay-group (future (summarise-results roll-up-total-in-send-by-ay-group simulations))
```

Comment:
Any reason this was moved?

Reply:
just to group things together by what I think will be the namespaces